src/foam/u2/wizard/data/EasyLoader.js

```diff
@@ -25,7 +25,10 @@ foam.CLASS({
 
   methods: [
     async function load({ old }) {
-      return await this.loaders[this.loaders.length - 1].load(...arguments);
+      for ( let i = 0 ; i < this.loaders.length ; i++ ) {
+        old = await this.loaders[i].load({ old });
+      }
+      return old;
```

Comment:
How is this different from the old one?

Reply:
if you keep assigning to old then it will overwrite the value every time the loop runs which means it will end up with the last iteration anyway